src/lib/holocene/checkbox.svelte

```diff
@@ -95,9 +95,11 @@
       {/if}
     </span>
 
-    <span class="label" class:sr-only={labelHidden}>
-      {label}
-    </span>
+    <slot name="label">
+      <span class="label" class:sr-only={labelHidden}>
```

Comment:
Could a checkbox potentially have both a label slot and a label prop? Wondering if we want to make this a conditional so that there's no potential for both.